dist/assets/c0318f8b.app.min.js

```diff
@@ -0,0 +1 @@
")[0],this.stripImageElement.style.width=""+this.frameWidth+"px",this.stripImageElement.style.height=""+this.frameHeight*this.framesCount+"px"},a.prototype.currentFrame=function(){return this.frames[this.frameIndex]},a.prototype.moveFrameToNext=function(){return this.frameIndex++},a.prototype.isAtLastFrame=function(){return this.frameIndex===this.framesCount-1},a.prototype.stripImageDidLoad=function(){},a.prototype.loadStripImage=function(){return this.stripImageElement.addEventListener("load",this.stripImageDidLoad,!1),this.stripImageElement.src=this.image_uri},a.prototype.activate=function(){return this.element.classList.remove("hide")},a.prototype.deactivate=function(){return this.element.classList.add("hide")},a.prototype.moveToCurrentFrame=function(){return this.currentFrame()||this.initializeFrames(),this.element.style.top=""+-1*this.currentFrame().topPosition+"px",this.element.style.left=""+-1*this.currentFrame().leftPosition+"px"},a.prototype.resetFrame=function(){return this.frameIndex=0,this.moveToCurrentFrame()},a}(),b["default"]=c}),define("multiSceneMovie/vastRequest",["multiSceneMovie/beacon","exports"],function(a,b){"use strict";var c,d=a["default"],e=function(a,b){return function(){return a.apply(b,arguments)}};c=function(){function a(a,b,c){this.url=a,this.eventEmitter=b,this.callback=c,this.setTrackers=e(this.setTrackers,this),this.clickThrough=void 0,this.ready=void 0,this.url&&this.run()}var b;return a.prototype.valid=function(){return this.url?!0:!1},a.prototype.run=function(){return this.requestHandler(this.setTrackers)},a.prototype.setTrackers=function(a){var c,d,e,f;if(a){for(d=[],Array.prototype.forEach.call(a.querySelectorAll("Linear VideoClicks ClickThrough"),function(a){return d.push({event:"clickThrough",url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Linear TrackingEvents Tracking"),function(a){return d.push({event:a.getAttribute("event"),url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Impression"),function(a){return d.push({event:"impression",url:a.textContent,type:"img"})}),e=0,f=d.length;f>e;e++)c=d[e],b(this.eventEmitter,c);return callback?this.callback():void 0}},b=function(a,b){var c;if(a){switch(c=void 0,b.event){case"creativeView":c="movie:screen:appeared";break;case"impression":c="movie:started";break;case"firstQuartile":c="movie:played:firstQuartile";break;case"midpoint":c="movie:played:half";break;case"thirdQuartile":c="movie:played:thirdQuartile";break;case"complete":c="movie:finished";break;case"clickThrough":c="movie:clickThrough"}return c?a.listen(c,function(){return new d(b.url,b.type)}):void 0}},a.prototype.requestHandler=function(a){var b;this.xhr||(this.xhr=new XMLHttpRequest),this.xhr.addEventListener("load",function(b){return function(){return a(b.xhr.responseXML),b.xhr.removeEventListener("load",arguments,!1),b.ready=!0}}(this),!1);try{return this.xhr.open("GET",this.url),this.xhr.send(null)}catch(c){b=c}},a}(),b["default"]=c});
```

Comment:
Line is too long.<br>'a' is already defined.<br>Possible strict violation.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>A constructor name should start with an uppercase letter.<br>Expected '{' and instead saw 'throw'.<br>Expected '{' and instead saw 'return'.<br>Too many errors. (100% scanned).